CONTRIBUTING.md

```diff
@@ -23,6 +49,81 @@ For feature requests, please:
 - Explain your motivation for the feature
 - List any alternatives you've considered
 
+## Branch Rules
+
+- `main` - Production-ready code, protected branch
+- `develop` - Integration branch for features
+- `feature/*` - New features
+- `release/*` - Release preparation
```

Comment:
if you need to def, then it means your naming needs improvement. hot fix branches are associated with bugs not with release. 

We can call the Emergency release branch to ER branch. (ER - Emergency Release) these release increment of patch version

Reply:
when a bug in found, then we figure out what it affects. applying that change to the develop branch would be labeled as a bug, while applying to an existing release would be called an emergency release instead of hotfix?

I am fine with that if you feel strongly, just seems like more words to me.